virtual_labs/external/nexus/project_interface.py

```diff
@@ -281,6 +283,32 @@ async def create_es_view(
                 type=NexusErrorValue.CREATE_ES_VIEW_ERROR,
             )
 
+    async def get_sbo_suite_views(self) -> list[ProjectView]:
```

Comment:
not important: if the `get_sbo_suite_views` not used/planned to be used ouside of this class, make it `private` make more sense, but not that important

Reply:
Thanks @bilalesi That's a good point. I've changed this. I'll wait for nexus to resolve the issues with ES views creation and then merge this PR.